tests/manage/z_cluster/test_osd_heap_profile.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import pytest
+import time
+
+from ocs_ci.framework.testlib import ManageTest, tier1, skipif_ocs_version
+from ocs_ci.utility.utils import ceph_health_check
+from ocs_ci.ocs.resources.pod import get_ceph_tools_pod, get_osd_pods, get_osd_pod_id
+
+log = logging.getLogger(__name__)
+
+
+@tier1
```

Comment:
this test case does not validate the basic functionality of a feature so I think it should not be part of tier1.

Reply:
@ebenahar how to mark this test ? tire4a?

Reply:
You can add it as part of tier2

Reply:
done